github/downloader_test.go

```diff
@@ -95,6 +95,9 @@ func isOSXOnTravis() bool {
 
 // Testing connection documentation, docker-compose and Migrate method
 func getDB(t *testing.T) (db *sql.DB) {
+	require.NotEmpty(t, os.Getenv("PSQL_USER"), "PSQL_USER env var not set")
+	require.NotEmpty(t, os.Getenv("PSQL_PWD"), "PSQL_PWD env var not set")
```

Comment:
pg password is optional. For example, I won't be able to run tests with default pg configuration from homebrew if we apply this change.